### PR TITLE
Improve unit test of MySQLInventoryDumper

### LINKED ISSUE #15707: [OpenForce 2022] Add unit test for MySQLInventoryDumper
Hi community,

This issue is for [OpenForce 2022](https://github.com/apache/shardingsphere/discussions/12683#discussioncomment-2139441)

### Aim
Add unit test for `MySQLInventoryDumper` in `shardingsphere-data-pipeline-mysql` module, doesn't include parent class methods.

### Basic Qualifications
- Java
- Maven
- Junit.Test

### PR DESCRIPTION
**Pull request by tain198127**
Fixes #15707.

Changes proposed in this pull request:
- rename MySQLJdbcDumperTest to MySQLInventoryDumperTest
- add more test case for YEAR_DATE_TYPE in MySQLInventoryDumper

### CODE REVIEW

**Comment by tain198127** [comment]
image
image
before I commit, Line coverage(the third column) was not 100% yet.

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/MySQLInventoryDumperTest.java [hunk 1]
```diff
@@ -30,24 +39,16 @@
...
 import static org.mockito.Mockito.when;
 
 @RunWith(MockitoJUnitRunner.class)
-public final class MySQLJdbcDumperTest {
-    
+public final class MySQLInventoryDumperTest {
+
     private MySQLInventoryDumper mysqlJdbcDumper;
-    
+
```
Comment (sandynz) [inline shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/MySQLInventoryDumperTest.java:51]: Whitespaces should be kept.
You could download and setup code style in IDE.

**Review by sandynz — CHANGES_REQUESTED** [review]

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/MySQLInventoryDumperTest.java (hunk 1 above)
Comment (tain198127) [reply shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/test/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/MySQLInventoryDumperTest.java]: fix it last commit.

**Review by sandynz — APPROVED** [review]

**Comment by sandynz** [comment]
@tain198127 Thanks for your contribution.

**Pull request merged**